src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
```

Comment:
pretty cool @magicmatatjahu 💯 💯 
Will Studio have Google Analytics? Perhaps we could send this info to analytics too (https://www.npmjs.com/package/web-vitals#user-content-send-the-results-to-google-analytics), does it make sense to do so?
Just asking because I think I saw some discussion on getting analytics from the playground, so Studio could have it too 🙂

What do you think?

Reply:
The file was created with `create-react-app` script, so it was added automatically. in future we will go with this as well as google analytics, but not now. Right now it's more important to get the Studio up and running. Google analytics and other things later :)